woocommerce/woocommerce
# Fix: variations are counted as products in import message of number of products imported

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Added the variations import count in the request response arrays and display in the message.

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a site using Ninja tool.
2. Install all the plugins.
3. Tap on "Run the Setup wizard".
4. Complete all the introductory tabs i.e., Store Details, Industry, Product types, Business details and Theme.
5. Go to dashboard.
6. Import a sample_products.csv file.
7. After completion of import it should show **18 products imported. 7 variations imported**. ( Instead of 25 products imported successfully )
